features/migrate_client_to_minion.feature

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2017 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Migrate a traditional client into a salt minion
+
+  Scenario: Migrate a sles client into a salt minion
+     Given I am authorized
+     When I follow "Salt"
+     Then I should see a "Bootstrapping" text
+     And I follow "Bootstrapping"
+     Then I should see a "Bootstrap Minions" text
```

Comment:
the ``` then ```we can remove it, keep it simple, since we are not testing the page, we want only to boostrap